aroquev/src/main/webapp/index.html

```diff
@@ -8,11 +8,90 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Armando Roque's Portfolio</h1>
+      <p>Hi and welcome to my personal portfolio! I am Armando, and here you will find interesting things about me (thanks for being interested!). I am currently studying Computer Science
+      at Tec de Monterrey. During the Summer of 2020 I will be working as a Google intern in the STEP program (this is where this project was born), really exciting stuff!</p>
+      <img id="main-photo" src="images/my-photo.JPG" alt="A photo of Armando">
+
+			<hr>
```

Comment:
nit: Can you look into reformatting this HTML file? Maybe with 2-space tabs?

Reply:
Done.

Reply:
Seems like the <hr> above is kind of hanging out and messing with indenting for the rest of the file?

Reply:
Yep, just fixed that in the latest commit.

Reply:
Oh no, you are right. Why does indentation appear different on github?